src/components/AdminPane/Manage/TaskAnalysisTable/TaskAnalysisTable.js

```diff
@@ -10,6 +10,8 @@ import _isObject from 'lodash/isObject'
 import _kebabCase from 'lodash/kebabCase'
 import _isUndefined from 'lodash/isUndefined'
 import _isFinite from 'lodash/isFinite'
+import parse from 'date-fns/parse'
+import { differenceInSeconds } from 'date-fns'
```

Comment:
Can you `import differenceInSeconds from 'date-fns/difference_in_seconds'` instead?